Backpack/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -44,6 +44,7 @@ data class CalendarParams(
   val locale: Locale = Locale.getDefault(),
   val dayOfWeekText: TextStyle = TextStyle.NARROW,
   val now: LocalDate = LocalDate.now(),
+  val wholeMonthSelectionLabel: String = "",
```

Comment:
Maybe this could be nullable instead? That way it's more clear that this is optional, rather than relying on a less-explicit empty string